index.js

```diff
@@ -95,7 +96,12 @@
 
       const startTime = process.hrtime();
       if (req.path === config.path) {
-        res.sendFile(path.join(__dirname + '/index.html'));
+        fs.readFile(path.join(__dirname + '/index.html'), function(err, content) {
+          content = content.toString().replace(new RegExp(defaultConfig.title, 'g'), config.title);
+          res.writeHead(200, {'Content-Type': 'text/html'});
+          res.write(content);
+          res.end();
+        });
```

Comment:
I think it would be better idea to generate temporary `index.rendered.html` on start of application instead of generating in on the fly everytime uses accesses `/status` page. 

I experimented with something like this here:
https://github.com/RafalWilinski/express-status-monitor/commit/d30faec1362bb55d20a07922c03fa6a3e6cb0d3c